main.go

```diff
@@ -191,6 +215,98 @@ func tryRebalance(ctx context.Context, r *regolancer, attempt *int) (err error,
 	return nil, true
 }
 
+func tryRapidRebalance(ctx context.Context, r *regolancer, from, to uint64, route *lnrpc.Route, amt int64) (successfullAtempts int, err error) {
+
+	rapidAttempt := 1
```

Comment:
A nitpick but I'd set it to 0 and only add 1 where it's needed. In most places below you subtract one. Not a strong opinion, I see that it makes sense to start with 1.

Reply:
is more elegant I guess